### ISSUE TITLE
[Feature Request] Add web shortcuts to App Drawer

### ISSUE THREAD

**Issue by bigrjsuto** [NeoApplications/Neo-Launcher#469]
## Description

Ability to add web shortcuts to App Drawer by drag 'n drop or via the Folders settings in Lawnchair.

So many apps aren't needed to be installed and web shortcuts are more than enough. Many apps I need less than once a month and don't want then adding bloat to the phone.

At the same time I love a minimalist homescreen and don't want anything on my homescreen except the dock and the "At a Glance" widget

## Expected Behavior
NA

## Actual Behavior
NA

## Steps to Reproduce
NA

## Device and OS
* Device: OnePlus 7 Pro
* Android version: 14
* Launcher version: 0.9.3
* Rom: crdroid

**Comment by avagraphique** [comment]
I agree this would be a nice feature. I use Hermit, which is expressly made for this purpose. 

As a workaround to keep my home screen tidy, I use folders, which isn't perfect but works.
